sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -222,6 +222,7 @@ export class MetricsBatchQueryClient {
 // @public
 export interface MetricsBatchQueryClientOptions extends CommonClientOptions {
     batchEndPoint?: string;
+    batchMetricsScope?: string;
```

Comment:
Consider naming this `batchMetricsAuthScope` to indicate that this is the scope used for auth.

Reply:
Done